streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -433,6 +485,8 @@
      * @param valueSerde        value serdes for materializing the aggregated table,
      *                          if not specified the default serdes defined in the configs will be used
      * @param name              the name of the resulted {@link KTable}
+     *
+     * @return a {@link KTable} that contains records with unmodified keys and values that represent the latest (rolling) aggregate for each key
```

Comment:
Isn't a windowed aggregate not different from a rolling aggregate? And if they are similar, why do we have both methods?

A windowed aggregate only considers the records contained in the window, while an rolling aggregate considers all records it processed so far (ie, this is a land-mark window, with fixed start time and end-time == infinite). Or do I miss understand something?